### DISCUSSION TITLE
Dynamic custom context injection

### DISCUSSION THREAD

**Discussion by scott-wright** [Netflix/dgs-framework#86] [Q&A] [answered]
I have a use case where I want to pass HTTP header data from clients of my DGS endpoint to a REST endpoint that I access on behalf of my clients.  I know I can do that in the graphql-java-spring-webmvc framework by creating a class that extends the GraphQLInvocation interface and supplying code that injects the headers from the request into the context it passes on to the data fetchers.  Is there a way to do that in the DGS framework?

**Comment by maxixcom** [comment, marked as answer]
Looks like this #67 could help here
